shared_kv_storage/db_store/db_store.h

```diff
@@ -93,5 +93,65 @@ void db_store_del(
     const std::string& table_b64,
     const std::string& key_b64,
     const std::string& value_b64);
+
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+/**
```

Comment:
Create a class for db_store with basic APIs get, set, lookup, remove and extend this class for CSV apis.